scripts/apps/ingest/services/SendService.js

```diff
@@ -174,6 +174,7 @@ export function SendService(desks, api, $q, notify, $injector, multi, $rootScope
         self.config = $q.defer();
         self.config.action = action;
         self.config.itemIds = _.map(items, '_id');
+        self.config.isPackage = _.some(items, (item) => item.type === 'composite');
```

Comment:
Use native [`Array.prototype.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

Reply:
All over the file is used lodash so I think we can keep it for consistency

Reply:
In a lot of cases, it's used for what no native methods are available like `cloneDeep`, `isEmpty`, `get`, `trim`. It's good that `lodash` is used there, but let's not use it for things that are already in the language. We'll never get rid of old stuff if we keep adding it. I've fixed that for you :)

Reply:
Please don't do it again. You just restarted my tests :)
There are many places in the application where we use lodash even if now there are native functions. For example there is a map call in the same file.
Probably a better approach is to change all those appearance and not to mixt them.

Reply:
> Please don't do it again. You just restarted my tests :)

I would like to do it more actually(commit to repos I'm reviewing, not restart your tests ;D) It's up to you of course, but I think it'd be more time efficient to resolve PRs when reviewers add commits for renames and other small stuff. What do you think?

> Probably a better approach is to change all those appearance and not to mixt them.

Yep, it's better to change it all, but they don't all have the same APIs and it's not always easy to figure out which types are used. For example `_.includes(scope.config.itemIds, data.item)`. I've no idea if `itemIds` is a delimited string or an aray or if `data.item` is string or an object. Since it's not some complex architectural pattern, but a simple pure function, I prefer mixing them so the amount of the old ones decreases and it's easier to remove them completely at some point.

Reply:
>I would like to do it more actually(commit to repos I'm reviewing, not restart your tests ;D) It's up to you of course, but I think it'd be more time efficient to resolve PRs when reviewers add commits for renames and other small stuff. What do you think?

I think it is better to just add comments.